src/handler.ts

```diff
@@ -519,6 +519,7 @@ export namespace Scheduler {
     metadata: { [key: string]: string };
     compute_types: string[] | null;
     default_compute_type: string | null;
+    utc_only: boolean | null;
```

Comment:
I don't think `null` is a possible value for the backend response, since there's no `null` equivalent in Python. You should be free to remove this and the weird `?? undefined;` clause in `create-job.tsx`.